designs/power-monitor.stanza

```diff
@@ -0,0 +1,64 @@
+#use-added-syntax(jitx)
```

Comment:
Add a header block saying what is is what it does, what it connects to. Note that some checks are expected to fail.